stuff/lib/db.py

```diff
@@ -20,9 +20,9 @@
 class DatabaseConnection(NamedTuple):
```

Comment:
un nit, esto no representa una conexión a la base de datos, sino los argumentos.

```suggestion
class DatabaseConnectionArguments(NamedTuple):
```

Reply:
Listo